### PR TITLE
flake8 and black fixes

### PR DESCRIPTION
**Pull request by goneri**
Ensure `tox` runs fine without any error. I'm not sure why Travis didn't
report the problem.

### CODE REVIEW

File: virt_lightning/configuration.py [hunk 1]
```diff
@@ -66,10 +69,7 @@ def __get(self, key):
 
     @property
     def libvirt_uri(self):
-        try:
-            return self.__get("libvirt_uri")
-        except:
-            pass
+        return self.__get("libvirt_uri")
```
Comment (zloidemon) [inline virt_lightning/configuration.py]: Code below will never been executed. Rollback or update.

File: virt_lightning/shell.py [hunk 2]
```diff
@@ -273,9 +272,9 @@ def main():
     except FileNotFoundError:
         print("Configuration file not found")
         exit(1)
-    except:
+    except Exception as e:
```
Comment (zloidemon) [inline virt_lightning/shell.py]: flake might fail it because variable has so short name.

Comment (goneri) [reply virt_lightning/shell.py]: It passes just fine.

File: virt_lightning/configuration.py (hunk 1 above)
Comment (goneri) [reply virt_lightning/configuration.py]: Will never be executed?

Comment (zloidemon) [reply virt_lightning/configuration.py]: Yes. You didn't delete check user:

screen shot 2019-02-15 at 2 11 22 am

Comment (goneri) [reply virt_lightning/configuration.py]: Aaah. It should be better with: https://github.com/virt-lightning/virt-lightning/commit/450b91c9556cef2e369f8f77f434b86134ef448e

**Pull request merged**